google/cloud/spanner/row.h

```diff
@@ -467,9 +468,11 @@ template <typename RowRange>
 auto GetSingularRow(RowRange range) -> std::decay_t<decltype(*range.begin())> {
   auto const e = range.end();
   auto it = range.begin();
-  if (it == e) return Status(StatusCode::kInvalidArgument, "no rows");
+  if (it == e)
+    return internal::InvalidArgumentError("no rows", GCP_ERROR_INFO());
   auto row = std::move(*it);
-  if (++it != e) return Status(StatusCode::kInvalidArgument, "too many rows");
+  if (++it != e)
+    return internal::InvalidArgumentError("too many rows", GCP_ERROR_INFO());
```

Comment:
We should add braces now that we have line breaks.

Reply:
DOne